src/fsharp/FSharp.Core/seq.fsi

```diff
@@ -243,6 +243,23 @@ namespace Microsoft.FSharp.Collections
         [<CompiledName("Empty")>]
         val empty<'T> : seq<'T>
 
+        /// <summary>Produces the set difference of two sequences by using generic hash and equality comparisons to compare values.</summary>
```

Comment:
I just cogged the [LINQ documentation](https://msdn.microsoft.com/en-us/LIBRARY/bb300779(v=vs.100).aspx)